tests/common/include/test_helpers.h

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test helpers for writing tests with standard interfaces.
+ *
+ * @author      Kevin Weiss <kevin.weiss@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include "shell.h"
+
+#ifndef TEST_HELPERS_H
+#define TEST_HELPERS_H
+
+#define JSON_SHELL_PARSER /**< If define parse will be json based */
```

Comment:
I guess this should go into the Makefile, i.e. `CFLAGS += -DJSON_SHELL_PARSER` - otherwise the next `ifdef` seems kind of weird.